tests/test_selection/test_smart_correlation_selection.py

```diff
@@ -267,7 +266,186 @@ def test_error_method_supplied(df_test):
     exceptionmsg = errmsg.value.args[0]
 
     assert (
-        exceptionmsg
-        == "method must be either 'pearson', 'spearman', 'kendall', or a callable,"
-        + f" '{method}' was supplied"
+            exceptionmsg
+            == "method must be either 'pearson', 'spearman', 'kendall', or a callable,"
+            + f" '{method}' was supplied"
     )
+
```

Comment:
General suggestion: you can use parametrisation for this tests :-) 

Then the code will be much readable :-D